src/spd-notification-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/spd/notification/worker/OutputNotificationConsumer.java

```diff
@@ -14,13 +31,138 @@
+                        .withApplicationPaymentId(documentData.getPvApplicationPaymentId())
+                        .withApplicationIncompleteReason(documentData.getPvApplicationIncompleteReason())
+                        .withApplicationValidateUsername(documentData.getPvValidationUser())
+                        .withApplicationDocumentGuid(documentResponse.getGuid())
+                        .withApplPartyId(documentData.getPnApplPartyId())
+                        .withApplSurname(documentData.getPvApplSurname())
+                        .withApplFirstName(documentData.getPvApplFirstName())
+                        .withApplSecondName(documentData.getPvApplSecondName())
+                        .withApplBirthDate(documentData.getPvApplBirthDate())
+                        .withApplGender(documentData.getPvApplGenderTxt())
+                        .withApplBirthPlace(documentData.getPvApplBirthPlaceTxt())
+                        .withApplAddlSurname1(documentData.getPvApplAddlSurname1())
+                        .withApplAddlFirstName1(documentData.getPvApplAddlFirstName1())
+                        .withApplAddlSecondName1(documentData.getPvApplAddlSecondName1())
+                        .withApplAddlSurname2(documentData.getPvApplAddlSurname2())
+                        .withApplAddlFirstName2(documentData.getPvApplAddlFirstName2())
+                        .withApplAddlSecondName2(documentData.getPvApplAddlSecondName2())
+                        .withApplAddlSurname3(documentData.getPvApplAddlSurname3())
+                        .withApplAddlFirstName3(documentData.getPvApplAddlFirstName3())
+                        .withApplAddlSecondName3(documentData.getPvApplAddlSecondName3())
+                        .withApplStreetAddress(documentData.getPvApplStreetAddress())
+                        .withApplCity(documentData.getPvApplCityNm())
+                        .withApplProvince(documentData.getPvApplProvinceNm())
+                        .withApplCountry(documentData.getPvApplCountryNm())
+                        .withApplPostalCode(documentData.getPvApplPostalCode())
+                        .withApplDriversLicence(documentData.getPvApplDriversLicence())
+                        .withApplPhoneNumber(documentData.getPvApplPhoneNumber())
+                        .withApplEmailAddress(documentData.getPvApplEmailAddress())
+                        .withApplOrgPartyId(documentData.getPnOrgPartyId())
+                        .withApplOrgFacilityPartyId(documentData.getPnOrgFacilityPartyId())
+                        .withApplOrgFacilityName(documentData.getPvOrgFacilityName())
+                        .withApplOrgContactPartyId(documentData.getPnOrgContactPartyId())
+                        .build();
+
+                DpsDataIntoFigaroResponse figaroResponse =
+                        documentService.dpsDataIntoFigaro(dpsDataIntoFigaroRequestBody);
+                logger.info("dpsDataIntoFigaroResponse: {}", figaroResponse.toString());
```

Comment:
Can you remove this log, it would put the body in the logs and replace with success on the operation